_src/static/sass/utilities/_variables.scss

```diff
@@ -362,6 +363,12 @@ $headings-primary-color: palette(grayscale, x-dark);
 $headings-secondary-font-family: $font-family-sans-serif;
 $headings-secondary-color: palette(grayscale, base);
 
+// buttons
```

Comment:
This is a good start to theming settings, but I don't see any of the following:

* text color (inactive, focus/hover, active, disabled)
* background color (inactive, focus/hover, active, disabled)
* settings for specific levels of buttons